.promu.yml

```diff
@@ -0,0 +1,48 @@
+go: 1.5.4
```

Comment:
As seen on IRC, we will check that later on, when another language needs to be handled by promu.